bygg-en-enkel-bot-main/saturday.js

```diff
@@ -0,0 +1,59 @@
+// Här är filen du ska jobba i! En funktion ska svara på
+// om det är lördag idag eller inte och en funktion ska
+// svara på hur många dagar det är kvar.
+
+// Den här funktionen ska returnera true om det är lördag
+// och false om det inte är lördag
+
+
+const d = new Date();
+const x = d.getDay();
+
+const dagar = [
```

Comment:
A good practice is to use english when creating variabels and functions. i.e use days instead of dagar 😃